src/util/BrowserFeatures.js

```diff
@@ -58,6 +58,14 @@ define(['underscore'], function (_) {
     return /(msie|trident)/i.test(navigator.userAgent);
   };
 
+  fn.isFirefox = function () {
+    return navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
+  };
+
+  fn.isChrome = function () {
+    return /Chrome/.test(navigator.userAgent) && /Google Inc/.test(navigator.vendor);
+  };
```

Comment:
Browser detection via userAgents is notoriously unreliable (see [here](http://webaim.org/blog/user-agent-string-history/) and the deprecated [jquery.browser](https://github.com/gabceb/jquery-browser-plugin/blob/master/dist/jquery.browser.js) plugin, which we've already removed from OSW).

Why can't we do feature detection?